packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -868,6 +880,132 @@ describe('postgresql-multi-schema', () => {
+    await expect(result).rejects.toThrowErrorMatchingInlineSnapshot(`
+
+      P4001 The introspected database was empty: postgres://prisma:prisma@localhost:5432/tests-migrate
+
+      prisma db pull could not create any models in your schema.prisma file and you will not be able to generate Prisma Client with the prisma generate command.
+
+      To fix this, you have two options:
+
+      - manually create a table in your database.
+      - make sure the database connection URL inside the datasource block in schema.prisma points to a database that is not empty (it must contain at least one table).
+
+      Then you can run prisma db pull again. 
+
+    `)
+    expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchSnapshot()
+    expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    expect(ctx.mocked['process.stdout.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    expect(ctx.mocked['process.stderr.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+  })
+
+  test('--url --schemas=base (1 existing schema) should succeed', async () => {
+    ctx.fixture('introspection/postgresql-multi-schema')
+
+    const introspect = new DbPull()
+    const result = introspect.parse(['--print', '--url', setupParams.connectionString, '--schemas', 'base'])
+    await expect(result).resolves.toMatchInlineSnapshot(``)
+    expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchSnapshot()
+    expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    expect(ctx.mocked['process.stdout.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    expect(ctx.mocked['process.stderr.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+  })
+
+  test('--url  --schemas=base,transactional (2 existing schemas) should succeed', async () => {
+    ctx.fixture('introspection/postgresql-multi-schema')
+
+    const introspect = new DbPull()
+    const result = introspect.parse([
+      '--print',
+      '--url',
+      setupParams.connectionString,
+      '--schemas',
+      'base,transactional',
+    ])
+    await expect(result).resolves.toMatchInlineSnapshot(``)
+    expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchSnapshot()
+    expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(`
+
+            // *** WARNING ***
+            // 
+            // These models and enums were renamed due to their names being duplicates in the Prisma Schema Language.
+            // - Enum "base_status"
+            // - Enum "transactional_status"
+            // - Model "base_some_table"
+            // - Model "transactional_some_table"
```

Comment:
Huh?

Reply:
Where can one see the SQL of this test to double check if there are indeed conflicts?

Reply:
Here, and it's expected https://github.com/prisma/prisma/blob/main/packages/migrate/src/__tests__/fixtures/introspection/postgresql-multi-schema/setup.sql